starter-code/index.html

```diff
@@ -1,31 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <section>
+  
+  <div class="box">
+    <!-- image iphone -->
+    <img src="./images/phones.png" alt="">
+  </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+  <div class="right-box box">
+    <!-- section droite -->
+    <header class="header-style">
+      <!-- header -->
+      <div>
+        <img src="./images/brand.png" alt="">
+      </div>
+      <button>
+        <img src="./images/home.png" alt="">
+        <a href="#0">Log in</a>
+      </button>
+    </header>
+    <div class="text-box">
+      <!-- titre et paragraphe -->
+      <h1>Capture and Share the World's Moments</h1>
+      <p>Instagram is a <b>fast, beautiful</b> and <b>fun</b> way to share your life with friends and family.</p>
+      <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+      <p>Oh yeah, did we mention it's free?</p>
+    </div>
+    <nav>
+      <!-- navigation bar -->
+      <a href="#0"><img src="./images/badge-iphone.png" alt=""></a>
+      <a href="#0"><img src="./images/badge-android.png" alt=""></a>
+    </nav>
+  </div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+</section>
 
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+<footer>
+  <a href="#0">ABOUT US</a>
+  <a href="#0">SUPPORT</a>
+  <a href="#0">BLOG</a>
+  <a href="#0">PRESS</a>
+  <a href="#0">API</a>
+  <a href="#0">JOBS</a>
+  <a href="#0">PRIVACY</a>
+  <a href="#0">TERMS</a>
+  <p>&copy; 2014 INSTAGRAM</p>
+</footer>
 </body>
```

Comment:
good use of the flex boxes, the code is clean, class used intelligently, I've got nothing to say! well done!